packages/devtools_flutter/lib/src/screen.dart

```diff
@@ -0,0 +1,30 @@
+import 'package:flutter/material.dart';
+
+class Screen extends StatelessWidget {
```

Comment:
Is each screen going to be a tab within the main application or are you envisioning a different setup?

Reply:
See the doc comments in https://github.com/flutter/devtools/pull/1106/files#diff-bcf4904b4fdc024eca0bbbe11b73ee4aR3